lms/envs/common.py

```diff
@@ -3374,7 +3386,18 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 FILE_UPLOAD_STORAGE_PREFIX = 'submissions_attachments'
 
 ##### ACCOUNT LOCKOUT DEFAULT PARAMETERS #####
+# .. setting_name: MAX_FAILED_LOGIN_ATTEMPTS_ALLOWED
+# .. setting_default: 6
+# .. setting_description: Specifies the maximum failed login attempts allowed to users. Once the user reaches this
+#   failure threshold then the account will be locked for a configurable amount of seconds (30 minutes) which will
+#   prevent additional login attempts until this time period has passed.
 MAX_FAILED_LOGIN_ATTEMPTS_ALLOWED = 6
```

Comment:
@jramnai just a recommendation, I think it would be better to also point here that this (and `MAX_FAILED_LOGIN_ATTEMPTS_LOCKOUT_PERIOD_SECS`) are related and only used when `ENABLE_MAX_FAILED_LOGIN_ATTEMPTS=True`, what do you think?

Reply:
Agreed @eLRuLL, I have updated the description of the setting.